src/components/CompareResults/landoToCommitLoader.ts

```diff
@@ -0,0 +1,66 @@
+import { checkValues, getComparisonInformation } from './loader';
+import { compareView } from '../../common/constants';
+import { fetchRevisionFromLandoId } from '../../logic/lando';
+import { Changeset, CompareResultsItem, Repository } from '../../types/state';
+import { Framework } from '../../types/types';
+
+// This function is responsible for fetching the data from the URL. It's called
+// by React Router DOM when the compare-hash-results path is requested.
+// This loader is used by ./mach try perf, and due to recent changes in
+// we now add the lando id. The lando id can be determined within about 30 seconds
+// of pushing to resolve to the treeherder commit
+export async function loader({ request }: { request: Request }) {
+  const url = new URL(request.url);
+  const baseLandoIDFromUrl = url.searchParams.get('baseLando');
+  const newLandoIDFromUrl = url.searchParams.get('newLando');
+  const baseRepoFromUrl = url.searchParams.get('baseRepo') as
+    | Repository['name']
+    | null;
+  const newReposFromUrl = url.searchParams.getAll(
+    'newRepo',
+  ) as Repository['name'][];
+  const frameworkFromUrl = url.searchParams.get('framework');
+  if (!baseLandoIDFromUrl || !newLandoIDFromUrl) {
+    throw new Error(
+      'Not all values were supplied please check you provided both baseLando and newLando',
+    );
+  }
+
+  const baseRevisionsFromLando =
+    await fetchRevisionFromLandoId(baseLandoIDFromUrl);
+  const newRevisionsFromLando =
+    await fetchRevisionFromLandoId(newLandoIDFromUrl);
+
+  const { baseRev, baseRepo, newRevs, newRepos, frameworkId, frameworkName } =
+    checkValues({
+      baseRev: baseRevisionsFromLando.commit_id,
+      baseRepo: baseRepoFromUrl,
+      newRevs: [newRevisionsFromLando.commit_id],
+      newRepos: newReposFromUrl,
```

Comment:
Is the `commit_id` from `baseRevisionsFromLando` the same as `baseRevision` from commits_from_hashes in `hashToCommitLoader.ts`? And if not, what's the difference?